index.js

```diff
@@ -100,10 +108,16 @@ function filterContributorByTime(idObject, moments) {
     const startMoment = moments[0];
     const endMoment = moments[1];
 
-    const timeWindow = moment.range([startMoment, endMoment]);
     for (let i = 0; i < idObject.contributions.length; i++) {
-        const momentOfContribution = moment.utc(idObject.contributions[i].created_at);
-        if (timeWindow.contains(momentOfContribution)) {
+        const momentOfContribution = createLuxonMomentFromIso(
+            idObject.contributions[i].created_at,
+            'Etc/UTC'
```

Comment:
is this correct `Etc/UTC`? I thought I understood from the README change above that that's not a valid Luxon timezone input.

Reply:
Turns out UTC is legal, and utc is legal at least on some systems, but maybe not others. Etc/UTC is not legal, as is Etc/GMT. Hopefully with my rewrites, users will have the information they need, and not too much.